src/core/link-to-dfn.js

```diff
@@ -26,6 +26,15 @@ const localizationStrings = {
     },
     duplicateTitle: "This is defined more than once in the document.",
   },
+  de: {
+    /**
+     * @param {string} title
+     */
+    duplicateMsg(title) {
+      return `Mehrfache Definition von '${title}'`;
+    },
+    duplicateTitle: "Das Dokument enthält mehrere Definitionen dieses Eintrags.",
```

Comment:
I don't know what is wrong here (the linter say `Insert '⏎·····'`).

Reply:
This line is long enough that prettier wants to insert a newline after the colon.

Reply:
Try running `npm run lint -- --fix`